example/src/app/simple/simple.component.spec.ts

```diff
@@ -36,3 +35,32 @@ describe('SimpleComponent', () => {
     expect(fixture.nativeElement).toMatchSnapshot();
   });
 });
+
+@Component({
+  selector: 'another-comp',
+  template: ''
+})
+class AnotherComponent {}
```

Comment:
Can we move this to another file? Until now we kept one component per file, like Angular recommends.

Reply:
Sure, I moved it to another file